core/src/main/scala/com/pingcap/tispark/examples/TiDataSourceExample.scala

```diff
@@ -202,12 +198,11 @@ object TiDataSourceExample {
                       |OPTIONS (
                       |  database 'tpch_test',
                       |  table 'target_table',
-                      |  tidb.addr '127.0.0.1',
+                      |  tidb.addr 'tidb',
```

Comment:
this look wired. Ang reason do this

Reply:
https://github.com/pingcap/tispark/pull/681